libs/wingsdk/test/target-sim/dynamodb-table.test.ts

```diff
@@ -146,12 +149,12 @@ test("update item", async () => {
         id: "S",
       },
       hashKey: "id",
+      hostPort: expect.any(String),
     },
     type: ex.DYNAMODB_TABLE_FQN,
   });
   await s.stop();
 
-  expect(listMessages(s)).toMatchSnapshot();
```

Comment:
intentional?

Reply:
yes, because now the messages have some non-deterministic output